rhizome/host/lib/vm_setup.rb

```diff
@@ -280,6 +280,7 @@ def setup_taps_6(gua, nics)
     end
 
     r "ip -n #{q_vm} addr replace fd00:0b1c:100d:5AFE:CE::/56 dev #{nics.first.tap}"
+    r "ip -n #{q_vm} addr replace fd00:0b1c:100d:53::/48 dev #{nics.first.tap}"
```

Comment:
I know it's kinda everywhere right now, but this whole `nics.first` business kinda worries me, I guess you're thinking it's just a dragon we'll have to slay later, in how to deal with multiple nics?

Reply:
For now, nics is a set with single element, the moment we introduce multiple nics, that will be a dragon to slay.